### PR TITLE
chore: converts one keyboard-language page to Markdown

### PR DESCRIPTION
**Pull request by jahorton**

### CODE REVIEW

File: developer/language/guide/comments.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Comments must be started with a ` c ` character (case-insensitive), and
+followed by a space character. The comment continues to the end of the
+line and cannot be
+[line-extended](long-lines "Long lines and continuations"). Comments
+can only be included in the last line of a
+[line-extended](long-lines "Long lines and continuations") line.
+
+## Example {#example}
+
+~~~ keyman
```
Comment (mcdurdin) [inline developer/language/guide/comments.md:14]:
```suggestion
```keyman
```

Elsewhere on the site we use triple-backtick, so it's better to continue with that

File: developer/language/guide/comments.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+c This is a comment line
++ 'a' > 'b'    c This line changes "a" to "b"
++ 'c' > \
+        'd'    c A comment here is okay too
+~~~
```
Comment (mcdurdin) [inline developer/language/guide/comments.md:19]:
````suggestion
```
````

File: developer/language/guide/comments.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: Comments
+---
+
+Comments must be started with a ` c ` character (case-insensitive), and
+followed by a space character. The comment continues to the end of the
+line and cannot be
+[line-extended](long-lines "Long lines and continuations"). Comments
```
Comment (ermshiperete) [inline developer/language/guide/comments.md:8]: The links don't look right.

File: developer/language/guide/comments.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+title: Comments
+---
+
+Comments must be started with a ` c ` character (case-insensitive), and
```
Comment (mcdurdin) [inline developer/language/guide/comments.md:5]:
```suggestion
Comments must be started with a `c` character (case-insensitive), and
```

File: developer/language/guide/comments.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+followed by a space character. The comment continues to the end of the
+line and cannot be
+[line-extended](long-lines "Long lines and continuations"). Comments
+can only be included in the last line of a
+[line-extended](long-lines "Long lines and continuations") line.
```
Comment (mcdurdin) [inline developer/language/guide/comments.md:10]: I think it's better to leave out the 'alt text' on the links as we can programmatically add it in the future at runtime

File: developer/language/guide/comments.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+## Example {#example}
```
Comment (mcdurdin) [inline developer/language/guide/comments.md:12]:
```suggestion
## Example
```

Don't include the anchor

File: developer/language/guide/comments.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+~~~ keyman
+c This is a comment line
++ 'a' > 'b'    c This line changes "a" to "b"
++ 'c' > \
+        'd'    c A comment here is okay too
+~~~
+
+## Version history {#version-history}
```
Comment (mcdurdin) [inline developer/language/guide/comments.md:21]:
```suggestion
## Version history
```

**Review by mcdurdin — CHANGES_REQUESTED** [review]

File: developer/language/guide/comments.md (hunk 5 above)
Comment (mcdurdin) [reply developer/language/guide/comments.md:10]:
```suggestion
[line-extended](long-lines). Comments can only be included in the last line of a
[line-extended](long-lines) line.
```

Comment (jahorton) [reply developer/language/guide/comments.md:10]: It was in the original, though... yeah, I was surprised to see it there.

File: developer/language/guide/comments.md (hunk 3 above)
Comment (jahorton) [reply developer/language/guide/comments.md]: That's a "link with alt text" format.  The original version's links had alt text specified too.

**Review by mcdurdin — APPROVED** [review]
LGTM

File: developer/language/guide/comments.md (hunk 5 above)
Comment (mcdurdin) [reply developer/language/guide/comments.md]: Some of the original content was converted from docbook, which did this kind of thing everywhere. Round and round we go!

**Pull request merged**